apps/crunchy-postgres/tests/node_affinity/run_litmus_test.yml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: litmus-node-affinity
+  namespace: litmus 
+spec:
+  template:
+    metadata:
+      name: litmus
+      labels:
+        app: node_affinity
+   
+    spec:
+      serviceAccountName: litmus
+      restartPolicy: Never
+      containers:
+      - name: ansibletest
+        image: openebs/ansible-runner:ci
+        imagePullPolicy: Always
+
+        env: 
+          - name: ANSIBLE_STDOUT_CALLBACK
+            #value: log_plays
+            value: default
+
+          - name: NAMESPACE
+            value: litmus
```

Comment:
@shashank855 use APP_NAMESPACE as a name

Reply:
@nsathyaseelan I have made the changes

Reply:
@shashank855 the application namespace will be `app-pgres-ns`

Reply:
@nsathyaseelan Sure !! I will make the changes.